package.json

```diff
@@ -17,7 +17,7 @@
     "lint:es": "node ../../scripts/eslint",
     "lint:style": "node ../../scripts/stylelint",
     "lint": "yarn run lint:es && yarn run lint:style",
-    "pretest:jest_server": "node ./test/jest_integration/runIdpServer.js &",
```

Comment:
This line is needed for the SAML selenium testing currently in this repo. The SAML selenium tests will be migrating to this repo (https://github.com/opensearch-project/opensearch-dashboards-functional-test) after the cypress upgrade from Cypress 9 to 12, but until then this line is necessary.

Reply:
Thank you for the observation @cwperks, resolved.